include/roboteam_ai/ApplicationManager.h

```diff
@@ -28,20 +28,25 @@ class ApplicationManager {
     bool fieldInitialized = false;
     bool robotsInitialized = false;
 
+    /**
+     * Current best play as picked by checker + decider
+     */
+    ai::stp::Play* bestPlay;
```

Comment:
Why raw pointer?

Reply:
Because the std::vector in PlayChecker owns the data (std::unique_ptr), and passing .get() is the easiest and cleanest way to use these values without copies. (T& would remove copy and move assignment operators)